### PR TITLE
ntp: display tweaks + stats show more fix

### PR DESCRIPTION
**Pull request by shakyShane**
**Asana Task/Github Issue:** https://app.asana.com/0/1201141132935289/1208864911781466/f
## Description

- [x] fixed an issue where the track-radar dataset has `Example.com`, but we only want to show 'Example'
- [x] fixed a bug where the `Show More` toggle would break if the list grew after initial expansion

## Testing Steps

- Check the 'Show More' toggle on this link https://deploy-preview-1288--content-scope-scripts.netlify.app/build/pages/new-tab/?stats=growing&stats-update-count=10

## Checklist

*Please tick all that apply:*

- [ ] I have tested this change locally
- [ ] I have tested this change locally in all supported browsers
- [ ] This change will be visible to users
- [ ] I have added automated tests that cover this change
- [ ] I have ensured the change is gated by config
- [ ] This change was covered by a ship review
- [ ] This change was covered by a tech design
- [ ] Any dependent config has been merged

### CODE REVIEW

**Comment by shakyShane** [comment]
* **#1288** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1288?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1288?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: special-pages/pages/new-tab/app/mock-transport.js [hunk 1]
```diff
@@ -256,30 +274,46 @@ export function mockTransport() {
...
+                            cb({
+                                trackerCompanies: list.slice(0, index + 1),
+                            });
+                            index++;
+                        }, 200);
+                        return () => {};
+                    } else {
+                        console.log(statsVariant);
+                        return () => {};
+                    }
```
Comment (shakyShane) [inline special-pages/pages/new-tab/app/mock-transport.js:316]: This is just about adding more scenarios for testing - none of these make it into the final release.

**Review by mgurgel — APPROVED** [review]
